content/en/05-reference/01-admin/09-tuning-mem-usage.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Tuning Memory Usage"
+metaTitle: "Reference | Admin | Tuning Memory Usage"
+metaDescription: "Tuning Pixie vizier-pem memory usage"
+order: 9
+---
+
+One of Pixie's unique features is that it collects, stores and queries all telemetry data *locally in your cluster*. Unlike other observability platforms, no telemetry data is transferred to a centralized backend outside of the cluster.
```

Comment:
I would drop Unlike other observability platforms but keep the rest of the sentence

Reply:
We may also want to add, "unless you choose to configure a plugin or export using the API" as a way to hype those features, and link out to them